### DISCUSSION TITLE
Actual state of the development

### DISCUSSION THREAD

**Discussion by ghost** [CoRE-RG/CoRE4INET#40] [General]
Hello there.

I am going to work with Real-Time networks simulations (RTP, RTPC, 802.1Q, ...), so I found CoRE4INET. I would like to ask, how active is the development of CoRE4INET? Are there plans to migrate to INET4.x?

Thanks!

**Comment by Musteblume** [comment]
Hi,

we are actively using and extending CoRE4INET for our research needs and publish all mature extensions. We are not in active development of INET4.x migration currently.

**Comment by ghost** [comment]
Thanks for reply. So, the most actual version that can (should) be used is *OMNeT++ 5.1.1 with the INET-Framework 3.5* as mentioned in [INSTALL](https://github.com/CoRE-RG/CoRE4INET/blob/master/INSTALL) file, right?

Thanks again.

**Comment by Musteblume** [comment]
Yes that is correct.

Best regards
